torch/distributions/transforms.py

```diff
@@ -61,12 +61,16 @@ class Transform(object):
             the codomain. Transforms that are not bijective should at least
             maintain the weaker pseudoinverse properties
             ``t(t.inv(t(x)) == t(x)`` and ``t.inv(t(t.inv(y))) == t.inv(y)``.
+        sign (int or Variable): For bijective univariate transforms, this
+            should be +1 or -1 depending on whether transform is monotone
+            increasing or decreasing.
         event_dim (int): Number of dimensions that are correlated together in
             the transform ``event_shape``. This should be 0 for pointwise
             transforms, 1 for transforms that act jointly on vectors, 2 for
             transforms that act jointly on matrices, etc.
     """
     bijective = False
+    sign = float('nan')
```

Comment:
This could be just `None`, correct? Do you have any use case of having `sign = nan`?

Reply:
Sure, it just needs to be some sort of undefined value. I guess it could be `None` or `float('nan')` or even
```py
@property
def sign(self):
    raise ValueError
```
I guess I chose `float('nan')` out of old C++ habits to make things of consistent type. Do you have a preference?